### PR TITLE
VIH-9534 Allow hearings to be booked on weekends

### PR DESCRIPTION
**Pull request by oliver-scott**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-9534

### Change description ###
Removes validation around the hearing dates being on a weekend

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite/Controllers/HearingsController.cs [hunk 1]
```diff
@@ -122,8 +122,8 @@ public async Task<IActionResult> CloneHearing(Guid hearingId, MultiHearingReques
         {
             _logger.LogDebug("Attempting to clone hearing {Hearing}", hearingId);
 
-            var hearingDates = hearingRequest.HearingDates != null && hearingRequest.HearingDates.Any() ? hearingRequest.HearingDates.Skip(1).ToList() : DateListMapper.GetListOfWorkingDates(hearingRequest.StartDate, hearingRequest.EndDate);
-
+            var hearingDates = GetDatesForClonedHearings(hearingRequest);
```
Comment (shaed-parkar) [inline AdminWebsite/AdminWebsite/Controllers/HearingsController.cs:125]: I'm a fan of any attempt to make this controller more readable!

File: AdminWebsite/AdminWebsite/Controllers/HearingsController.cs [hunk 2]
```diff
@@ -157,6 +157,21 @@ public async Task<IActionResult> CloneHearing(Guid hearingId, MultiHearingReques
             }
         }
 
+        private static List<DateTime> GetDatesForClonedHearings(MultiHearingRequest hearingRequest)
+        {
+            if (hearingRequest.HearingDates != null && hearingRequest.HearingDates.Any())
```
Comment (shaed-parkar) [inline AdminWebsite/AdminWebsite/Controllers/HearingsController.cs:162]: Is this to cover the scenario when VHOs provide a list of single dates?

**Review by shaed-parkar — APPROVED** [review]
LGTM

File: AdminWebsite/AdminWebsite/Controllers/HearingsController.cs (hunk 2 above)
Comment (oliver-scott) [reply AdminWebsite/AdminWebsite/Controllers/HearingsController.cs:162]: yes that's correct - HearingDates contains the individual list of dates

**Pull request merged**